gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -651,8 +649,91 @@ private void testFileChecksum(
     assertThat(fileChecksum.getAlgorithmName()).isEqualTo(checksumType.getAlgorithmName());
     assertThat(fileChecksum.getLength()).isEqualTo(checksumType.getByteLength());
     assertThat(fileChecksum.getBytes()).isEqualTo(checksumFn.apply(fileContent));
+    assertThat(fileChecksum.toString()).contains("%s: ".format(checksumType.getAlgorithmName()));
 
     // Cleanup.
-    assertThat(ghfs.delete(filePath, true)).isTrue();
+    assertThat(ghfs.delete(filePath, /* recursive= */ true)).isTrue();
+  }
+
+  @Test
+  public void testInitializeWithEmptyWorkingDirectory() throws IOException {
```

Comment:
Add description of the expected result to test name: `testInitializeWithEmptyWorkingDirectory_should...`